packages/core/useScroll/index.ts

```diff
@@ -182,7 +191,7 @@ export function useScroll(
     internalY.value = scrollTop
 
     isScrolling.value = true
-    onScrollEnd(e)
+    !useNativeScrollEnd && onScrollEndDebounced(e)
```

Comment:
I think we could both have:
add the none debounced  `onScrollEnd` on `scrollend` event,
and keep the `onScrollEndDebounced` in `onScrollHandler`.

So we can get scroll end state a little bit faster if `scrollend` event is supported, otherwise it works as normal as before. And no extra option is needed.

Reply:
But it may be invoke `onStop` callback twice, one by scrollend, the other by debounced handler. Is it expected?

Reply:
We could do something like:
```js
const onScrollEnd = (e: Event) => {
  // dedupe the second call
  if (!isScrolling.value) return

  isScrolling.value = false
  directions.left = false
  directions.right = false
  directions.top = false
  directions.bottom = false
  onStop(e)
}
```

Reply:
got it and done~